src/screens/Asset/ReceiveModal.js

```diff
@@ -186,9 +207,8 @@ const QRCodeWrapper = styled.View`
 `;
 
 const WalletAddress = styled(Text)`
```

Comment:
According to design wallet address should use text style `small` as its 12px font. Pls apply it to both ens & non-ens scenario.

Reply:
Done.